Paper Feedback (#192)
precice/aste: docs/paper.md

As additional software components, so-called adapters bridge the gap between the preCICE API and the software environments used by the coupled models.
Creating and using this overall setup for early development purposes is not only cumbersome, but also very inefficient.
The artificial solver testing environment (ASTE) allows to replace and imitate models coupled via preCICE by artificial ones, potentially in parallel distributed across multiple ranks on distributed memory.
This helps in the development of preCICE, adapters, or simulation setups by reducing the necessary software components, simplifying execution workflows, and reducing runtime of the case.
In addition, ASTE provides performance and accuracy metrics of the configured simulation setup.

# Statement of need

\autoref{fig:dependency-graph} illustrates the software stack required for a coupled simulation setup using FEniCS and OpenFOAM as examples, and compares it to a simulation setup using ASTE.
Beside preCICE itself, core ingredients for practical applications are preCICE API language bindings, preCICE adapters, the simulation frameworks and their dependencies.

> Note that the framework dependencies are usually heavy by themselves, e.g., packages such as Dune or Trilinos.

Author: I would void PETSc as it is a dependency of precice

Reviewer: PETSc is optional though. I put it because it's well known and established. If we put Dune, I assume we would have to cite it?

Author: Citations for dune https://www.dune-project.org/publications/dune-references.bib

We could also include packages like HyPr, ScaLAPACK, [SUNDIALS](https://computing.llnl.gov/projects/sundials), Kokkos, Ginkgo, Pforest.

Reviewer: ```suggestion
```

For the configuration of data mappings particularly, ASTE can evaluate additional accuracy metrics.
Combined with the performance instrumentation of preCICE itself, this enables finding appropriate settings for specific scenarios (e.g. as demonstrated in the large-scale example in @ExaFSA2020).

Lastly, ASTE provides a reproducible environment which enables to share and rerun scenarios, regardless of the availability of involved software components. This capability is particularly useful for debugging issues reported by users of preCICE, who can share their scenarios (e.g. through the [preCICE forum](https://precice.discourse.group/)) for developers to analyze, even in case the involved software is unavailable due to licensing terms or being closed-source.

# Functionality & Use

The central interface of ASTE is given through a VTK mesh file, which contains information about the geometric shape of the model we emulate.

> The VTK files can be generated from mesh generation tools (e.g., GMSH [@gmsh]), [Python scripts](https://github.com/precice/aste/tree/develop/tools/mesh-generators) or directly reused from [preCICE exports of a simulation](https://precice.org/configuration-export.html).

Reviewer: ```suggestion
The VTK files can be generated from mesh generation tools (e.g., GMSH [@gmsh]), [Python scripts](https://github.com/precice/aste/tree/develop/tools/mesh-generators) or directly reused from [preCICE simulation exports](https://precice.org/configuration-export.html).
```
```suggestion
The VTK files can be generated from mesh generation tools (e.g., GMSH [@gmsh]), [Python scripts](https://github.com/precice/aste/tree/develop/tools/mesh-generators) or directly reused from [preCICE](https://precice.org/configuration-export.html) after the simulation completed.
```
I am not sure to which extend the 'export' here is domain terminology and understandable. Is the suggestion about addinf the export or stressing that preCICE ran before?

Author: ```suggestion
The VTK files can be generated from mesh generation tools (e.g., GMSH [@gmsh]), [Python scripts](https://github.com/precice/aste/tree/develop/tools/mesh-generators) or directly reused from a [completed preCICE simulation](https://precice.org/configuration-export.html).
```
